airbyte-integrations/connectors/destination-bigquery/src/test-integration/java/io/airbyte/integrations/destination/bigquery/typing_deduping/BigQuerySqlGeneratorIntegrationTest.java

```diff
@@ -444,6 +445,16 @@ public void testFailureOnReservedColumnNamePrefix(final String prefix) {
     );
   }
 
+  /**
+   * Something about this test is borked on bigquery. It fails because the raw table doesn't exist,
+   * but you can go into the UI and see that it does exist.
+   */
+  @Override
+  @Disabled
+  public void noCrashOnSpecialCharacters(final String specialChars) throws Exception {
+    super.noCrashOnSpecialCharacters(specialChars);
+  }
+
```

Comment:
Is there a way to see the queries the UI is making, or use tablePlus or another SQL ui to see if they are doing something special to show you those reults?

Reply:
it's actually dumber than that. the insert query in the PR description fails because the raw table doesn't exist, but if you pull out the `select` thing and run that on its own, it succeeds. So this works:
```sql
SELECT 
  'arst',
  json'{}',
 _airbyte_raw_id,
 _airbyte_extracted_at
FROM `sql_generator_test_fvdrq__`.`raw_table`
```

![image](https://github.com/airbytehq/airbyte/assets/5741425/9f4344bf-7e52-40b6-a435-b9c96605319e)

but the original insert query fails

![image](https://github.com/airbytehq/airbyte/assets/5741425/9cf9d03d-4ef8-4b80-a3cb-5f62f2ed7edb)